scripts/doc_checker.py

```diff
@@ -255,6 +264,16 @@ def get_noqa_checks(doc: Docstring) -> list:
     If noqa doesn't have any codes - returns ["all"].
     """
     source = doc.method_source
+    # case when property decorator is used; it hides sources
+    if not source and isinstance(doc.obj, property):
+        # readonly property
```

Comment:
Why do we handle read-only different from read-write? I think a convention is to document everything under `fget` part

Reply:
Indeed, I did not see the difference.